main.py

```diff
@@ -448,6 +449,20 @@ async def list_result_directory() -> List[int]:
     return rid_list
 
 
+@app.get("/dataset/master/")
+async def get_master_dataset(key: str) -> Union[int, float, List]:
+    """Returns the dataset broadcast to artiq master.
+    
+    Args:
+        key: The key of the target dataset.
+    """
+    remote = get_client("master_dataset_db")
+    array = remote.get(key)
+    if isinstance(array, np.ndarray):
```

Comment:
It is just a question!
When isn't the `array` a numpy array? The case it is a number?

Reply:
The dataset in artiq does not restrict the type of the values in it. It can be anything in principle!

Reply:
Oh, I see! Thanks👍